docs/experiments-microblog.md

```diff
@@ -0,0 +1,44 @@
+# Anserini Experiments on TREC Microblog Collection
+
+
+### Tweets2011 collection
+
+Note that the Tweets2011 collection is distributed as a list of tweet ids that you have to download yourself, so the 
+effectiveness results you'll get should be similar, but will likely not be identical to the scores reported here.
+
+Indexing the Tweets2011 collection:
+
+```
+nohup sh target/appassembler/bin/IndexCollection -collection TwitterCollection  -input \
+/path/to/Tweets2011/ -generator JsoupGenerator  -index lucene-index.Tweets2011.pos+docvectors -threads 32 \
+-storePositions -storeDocvectors -optimize > log.Tweets2011.txt &
+```
```

Comment:
I remember a similar opinion from the previous CR too :P The documentation throughout Anserini follows a consistent with this notation, lest an unsuspecting user just copy-paste the command on his server. I can change it, if you insist!